documentation/modules/proc-installing-product.adoc

```diff
@@ -0,0 +1,73 @@
+// Module included in the following assemblies:
+//
+// assembly-evaluation.adoc
+
+[id='proc-install-product-{context}']
+= Installing {ProductName}
+
+Using the xref:proc-product-downloads-{context}[download files], install {ProductName} with the Custom Resource Definitions (CRDs) and RBAC configuration required for deployment.
+
+.Prerequisites
+
+* Installation requires a Kubernetes account with cluster admin credentials
+
+.Procedure
+
+. Log in to the Kubernetes cluster with cluster admin privileges.
+
+. Modify the installation files to reference a new `kafka` namespace where you will install the {ProductName} Kafka Cluster Operator.
++
+NOTE: By default, the files work in the `myproject` namespace.
++
+* On Linux, use:
++
+
+[source, shell, subs=+quotes]
+----
+sed -i 's/namespace: .\*/namespace: kafka/' install/cluster-operator/*RoleBinding*.yaml
+----
++
+* On Mac, use:
++
+[source, shell, subs=+quotes]
+----
+sed -i '' 's/namespace: .\*/namespace: kafka/' install/cluster-operator/*RoleBinding*.yaml
+----
+
+. Create the new `kafka` namespace.
++
+[source, shell, subs=+quotes ]
+----
+kubectl create ns kafka
+----
+
+. Deploy the CRDs and role-based access control (RBAC) resources to manage the CRDs.
++
+[source, shell, subs=+quotes ]
+----
+kubectl apply -f install/cluster-operator/ -n kafka
+----
+
+. Create the project `my-kafka-project` where you will deploy your Kafka cluster.
```

Comment:
Please remind me, why is it necessary to create this project after creating the `kafka' namespace?

Also, you use the term "project" but the command creates a namespace using the same command as in step 3. This could be confusing for beginners.